### ISSUE TITLE
Website redesign

### ISSUE THREAD

**Issue by noClaps** [quillpad/quillpad.github.io#1]
I was interested in the Quillpad project, so I went to your website, but it felt off, so I decided to redesign it. My new design is similar to the original one using the ReFresh theme, but I've written the whole thing from scratch. It's currently live at https://noClaps.github.io/quillpad-website.

If you want the website, I can transfer it over and you can configure it how you want. If there's any changes you want to discuss, you can open an issue on the repo: https://github.com/noClaps/quillpad-website.

Cheers!

**Comment by jsixface** [comment]
The current website is literally me downloading the theme, changing the name and adding a privacy policy page so that I can put the link in Google play store. 

Thank you for the offer on transfer, but I would like to have both the site and the app under the same org. So if you are okay with it, can you create a PR for this repo ?
And thank you for the effort.

**Comment by noClaps** [comment]
By transfer I meant transfer the repo over to the Quillpad org. You can then rename the repo to quillpad.github.io and change the config accordingly, and it would work exactly the same as the current one. I feel like that would be easier than creating a PR when I didn't fork this repo in the first place.

**Comment by jsixface** [comment]
Sounds good. Thank you.

**Comment by noClaps** [comment]
It says I don't have permission to create public repositories on quillpad, so I can't transfer it.

**Comment by jsixface** [comment]
I will fork the repo and rename it. 
I'm going to update the repo license to GPL (same as the app) to keep it open source. Hope that's okay.

**Comment by noClaps** [comment]
0BSD is an open source license too, but sure.

Anyway, feel free to close this issue once you're done migrating. Also you can open an issue or PR in my repo if you've got any changes you want implemented or design suggestions.

Cheers!

**Comment by jsixface** [comment]
Yeah. Bsd doesn't have conditions to release the derivative work as open source as well..


________________________________

**Comment by noClaps** [comment]
If that's something you want, then fair enough.
